packages/block-renderer/src/hooks/use-rendered-patterns.ts

```diff
@@ -56,7 +60,7 @@ const useRenderedPatterns = (
 				pages.reduce( ( previous, current ) => ( { ...previous, ...current } ), {} )
 			);
 		} );
-	}, [] );
+	}, [ patternIds ] );
```

Comment:
Before we had the `patternIds` ready when the component mounts because we used a static file but in this PR the patterns are fetched so we need to watch changes on `patternIds`.